### ISSUE TITLE
topDirs function isn't compatible with hic_glm when multiple contrasts are passed through

### ISSUE THREAD

**Issue by serine** [dozmorovlab/multiHiCcompare#8]
Hi @mdozmorov ,

I'm not 100%, but it looks like `topDirs` function isn't compatible with `hic_glm` if I pass multiple contrasts. 

```
design <- model.matrix(~0+groups)
colnames(design) <- gsub("groups", "", colnames(design))
rownames(design) <- meta$samples

contrasts <- limma::makeContrasts(mock1_vs_mock2=mock2-mock1,
                                  mock3_vs_mock4=mock4-mock3,
                                  levels = design)

hic_fit <- hic_glm(hic_exp,
                   design = design,
                   contrast = contrasts,
                   method = "QLFTest",
                   p.method = "fdr",
                   parallel = TRUE)
```

when I call `results(hic_fit)` I get nice table with all contrasts tested, but the columns labeled like this `logFC.mock1_vs_mock2` and `logFC.mock3_vs_mock4`. From what I can tell `topDirs` expects `logFC` column label [here](https://github.com/dozmorovlab/multiHiCcompare/blob/41d4d5030f1d4d1c736830a6a9e47d1a8b4ce1f7/R/topDirs.R#L73)

It isn't an issue for me, I can workaround by just using `results` function, but I thought it would be worth raising for future improvements. 

Thanks,

**Comment by mdozmorov** [comment]
Indeed, working with multiple contrasts would be good to implement. We haven't tested it that thoroughly. I'll keep this issue open, or @serine, please, make a pull request if you fix it.
